dymos/phase/options.py

```diff
@@ -656,3 +656,32 @@ def __init__(self, read_only=False):
         self.declare(name='flat_indices', types=bool, default=True,
                      desc='If True, the given indices will be treated as indices into a C-order flattened array based '
                           'on the shaped of the constrained variable at a point in time.')
+
+
+class TimeseriesOutputOptionsDictionary(om.OptionsDictionary):
+    """
+    An OptionsDictionary for timeseries outputs.
+
+    Parameters
+    ----------
+    read_only : bool
+        If True, setting (via __setitem__ or update) is not permitted.
+    """
+    def __init__(self, read_only=False):
+        super(TimeseriesOutputOptionsDictionary, self).__init__(read_only)
+
+        self.declare(name='name', types=str, default=None, allow_none=True,
+                     desc='Name or ODE-relative path of the variable to be constrained.')
```

Comment:
`name` and `shape` mention constraints. MIght be cut-and-paste error.

Reply:
Thanks, fixed.